src/dvclive/live.py

```diff
@@ -193,7 +193,7 @@ def _init_dvc_file(self) -> str:
                 return self._dvcyaml
             raise InvalidDvcyamlError
         if self._dvc_repo is not None:
-            return os.path.join(self._dvc_repo.root_dir, "dvc.yaml")
+            return os.path.relpath(os.path.join(self._dvc_repo.root_dir, "dvc.yaml"))
```

Comment:
whi is the `relpath` needed?

Reply:
Without it, we end up doing `git add /var/tmp/.../dvc.yaml`, which I guess is fine, so I can update the tests to expect that.